test/metabase/query_processor_test/date_bucketing_test.clj

```diff
@@ -899,9 +899,10 @@
 
 (defmethod mt/get-dataset-definition TimestampDatasetDef
```

Comment:
This method was updated so we are able to specify number of intervals to genrate.